packages/collections/graphql/resolvers/User.js

```diff
@@ -13,6 +13,22 @@ const canAccess = (user, context) => {
   )
 }
 
+const progressFinishedThreshold = 0.85
+const progressConditions = {
+  FINISHED: {
+    or: [
+      { "data->>'percentage' >=": progressFinishedThreshold },
+      { "data->>'percentage'": null },
```

Comment:
What is the motivation of selecting records `WHERE "percentage" IS NULL`?

Reply:
The null selector ensure compatibility with non-progress items. E.g. selecting `bookmark` (without a `data->>'percentage'`) and `progress` items that are unfinished.